arangod/IResearch/IResearchInvertedIndexMeta.cpp

```diff
@@ -409,46 +409,17 @@ bool IResearchInvertedIndexMeta::json(
 
 bool IResearchInvertedIndexMeta::operator==(
     IResearchInvertedIndexMeta const& other) const noexcept {
-  if (*static_cast<IResearchDataStoreMeta const*>(this) !=
-      static_cast<IResearchDataStoreMeta const&>(other)) {
-    return false;
-  }
-
-  if (_sort != other._sort) {
-    return false;
-  }
-
-  if (_consistency != other._consistency) {
-    return false;
-  }
-
-  if (_storedValues != other._storedValues) {
-    return false;
-  }
-
-  if (_version != other._version) {
-    return false;
-  }
-
-  if (_fields.size() != other._fields.size()) {
-    return false;
-  }
-
-  size_t matched{0};
-  for (auto const& thisField : _fields) {
-    for (auto const& otherField : _fields) {
-      if (thisField.namesMatch(otherField)) {
-        matched++;
-        break;
-      }
-    }
-  }
-  return matched == _fields.size();
+  return (*static_cast<IResearchDataStoreMeta const*>(this) ==
+          static_cast<IResearchDataStoreMeta const&>(other)) &&
+         (*static_cast<InvertedIndexField const*>(this) ==
```

Comment:
NIT, better format alignment
```suggestion
         (static_cast<InvertedIndexField const&>(*this) ==
```